scripts/dts/gen_defines.py

```diff
@@ -518,6 +533,14 @@ def write_phandle_val_list_entry(dev, entry, i, ident):
         if i is not None:
             cell_ident += "_{}".format(i)
         out_dev(dev, cell_ident, val)
+        initializer_vals.append(val)
+
+    initializer_ident = ident + "S"
+    if entry.name:
+        initializer_ident += "_" + str2ident(entry.name)
+    if i is not None:
+        initializer_ident += "_{}".format(i)
+    return out_dev(dev, initializer_ident, "{" + ", ".join(map(str, initializer_vals)) + "}")
```

Comment:
Could shorten the line a bit maybe:

```python
    return out_dev(dev, initializer_ident,
                   "{" + ", ".join(map(str, initializer_vals)) + "}")
```

Not that I'm super-obsessed with stuff like that, but looks pretty nice in that case.

Reply:
both done.